fe/fe-core/src/main/java/org/apache/doris/datasource/SplitAssignment.java

```diff
@@ -156,7 +156,7 @@ public BlockingQueue<Collection<TScanRangeLocations>> getAssignedSplits(Backend
         return splits;
     }
 
-    public void setException(UserException e) {
+    public synchronized void setException(UserException e) {
```

Comment:
Why using `synchronized `? add comment in code

Reply:
It seems that there is no need to add synchronized.